CodeStarterCamp_Week3/step2.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  step2.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by 홍세희 on 2023/09/12.
+//
+
+enum Coffee: String {
+    case americano = "아메리카노"
+    case cafeLatte = "카페라떼"
+    case cappuccino = "카푸치노"
+    case mocha = "모카"
+    case prapucihno = "프라푸치노"
+    case banillaLatte = "바닐라라떼"
+}
+
+struct Person {
+    var money: Int
+    var name: String
+
+    func order(_ coffee: Coffee, of coffeeShop: CoffeeShop, by name: String) {
+        if let price = menu[coffee] {
+            if money < price {
+                print("잔액이 \(price - money)원 만큼 부족합니다. ")
+            }
+            else {
```

Comment:
if-else 문 컨벤션은 보통 이렇습니다 👀
```suggestion
            if money < price {
                print("잔액이 \(price - money)원 만큼 부족합니다. ")
            } else {
```